FuzzTesting/Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.4
+// swift-tools-version:5.7
```

Comment:
Do we need this change?

Reply:
This change is okay: oss-fuzz recently updated their Swift image to be 5.7.